ReplaySample/src/main/java/com/foresee/ReplaySample/MaskingActivity.java

```diff
@@ -29,12 +31,17 @@ protected void onCreate(Bundle savedInstanceState) {
         editText = (EditText) findViewById(R.id.maskingText);
         imageView = (ImageView) findViewById(R.id.maskingImage);
 
+        maskingTextLabel = (TextView) findViewById(R.id.maskingTextLabel);
+        maskingImageLabel = (TextView) findViewById(R.id.maskingImageLabel);
+
         maskEditTextToggleButton.setOnCheckedChangeListener(new CompoundButton.OnCheckedChangeListener() {
             @Override
             public void onCheckedChanged(CompoundButton compoundButton, boolean b) {
                 if (maskEditTextToggleButton.isChecked()) {
+                    maskingTextLabel.setText("Text field masking: OFF");
                     Replay.unmaskView(editText);
                 } else {
+                    maskingTextLabel.setText("Text field masking: ON");
```

Comment:
Haha, you misunderstood what I meant... 

I was trying to say that the label should always read "Text field masking", be to the left of the button, and the button itself should provide the ON or OFF text. The state of the masking should also match up with the state of the button. Right now, the button says ON when the masking is disabled, which is confusing.